### DISCUSSION TITLE
Federation?

### DISCUSSION THREAD

**Discussion by EsmailELBoBDev2** [stoatchat/discussions#114] [Feature Suggestions]
Just like matrix. you already nailed the design of the app but still giving federation option would be a great idea so we all can contact together instead of creat account on each server

**Comment by sussycatgirl** [comment]
Not going to happen, sadly.

**Reply by oleggtro** [reply]
Would a PR adding it be accepted?

**Comment by oleggtro** [comment]
duplicate #103

**Reply by janwels89** [reply]
This is not a Dublicate -  #103 is about SSO and a centralized Authentication Server.

Federation means, at least for me, that Self-Hosted Instances would trust each other and users can join with their existing account.
